include/FreeRTOS.h

```diff
@@ -900,9 +900,9 @@
 
 #ifndef configSTACK_DEPTH_TYPE
 
-/* Defaults to uint16_t for backward compatibility, but can be overridden
+/* Defaults to uint32_t for compatibility, but can be overridden
  * in FreeRTOSConfig.h if uint16_t is too restrictive. */
-    #define configSTACK_DEPTH_TYPE    uint16_t
+    #define configSTACK_DEPTH_TYPE    uint32_t

```

Comment:
This constant is to ensure dropping the code into a a project that doesn't define configSTACK_DEPTH will work as per prior versions - so uint16_t was the correct value.  https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/V8.0.0/FreeRTOS/Source/tasks.c#L515

Reply:
Default reverted to `uint16_t`.